sdk/textanalytics/Azure.AI.TextAnalytics/src/TextAnalyticsClient.cs

```diff
@@ -2040,6 +2040,138 @@ public virtual async Task<AnalyzeHealthcareEntitiesOperation> StartAnalyzeHealth
 
         #endregion
 
+        #region Extract Summary
+
+        /// <summary>
+        /// Performs extractive summarization on the given documents, which consists of extracting
+        /// sentences that collectively represent the most important or relevant information within
+        /// the original content.
+        /// For a list of languages supported by this operation, see
+        /// <see href="https://learn.microsoft.com/en-us/azure/cognitive-services/language-service/summarization/language-support?tabs=document-summarization"/>.
+        /// For document length limits, maximum batch size, and supported text encoding, see
+        /// <see href="https://aka.ms/azsdk/textanalytics/data-limits"/>.
+        /// </summary>
+        /// <remarks>
+        /// This method is only available for <see cref="TextAnalyticsClientOptions.ServiceVersion.V2022_10_01_Preview"/>, and newer.
+        /// </remarks>
+        /// <param name="documents">The documents to analyze.</param>
+        /// <param name="language">The language that the document is written in.</param>
+        /// <param name="options">The additional configurable options<see cref="ExtractSummaryOptions"/></param>
+        /// <param name="cancellationToken">A <see cref="CancellationToken"/> controlling the request lifetime.</param>
+        /// <returns>
+        /// An <see cref="ExtractSummaryOperation"/> that can be used to monitor the status of this long-running operation.
+        /// Upon completion, the <see cref="ExtractSummaryOperation"/> will contain the collections of extracted summary
+        /// sentences for each document that was successfully analyzed.
+        /// </returns>
+        /// <exception cref="NotSupportedException">This method is only supported in service API version 2022-10-01-preview and newer.</exception>
+        /// <exception cref="RequestFailedException">Service returned a non-success status code.</exception>
```

Comment:
There are also `ArgumentException`s and derivatives that could throw. Be sure to document those as well.